From 1eb8fe02137b95f384508c676af20adaeac31971 Mon Sep 17 00:00:00 2001
From: wesaphzt <45947243+wesaphzt@users.noreply.github.com>
Date: Wed, 11 Sep 2019 11:39:29 +0000
Subject: [PATCH] Update AsyncTask.vala

Fix error when building from source
Utility/AsyncTask.vala:76.2-76.17: error: Creation method of abstract class cannot be public.
public AsyncTask(){
^^^^^^^^^^^^^^^^
---
 src/Utility/AsyncTask.vala | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Utility/AsyncTask.vala b/src/Utility/AsyncTask.vala
index f5504e8..0bbc85f 100644
--- src/Utility/AsyncTask.vala
+++ src/Utility/AsyncTask.vala
@@ -73,7 +73,7 @@ public abstract class AsyncTask : GLib.Object{
 	public signal void stderr_line_read(string line);
 	public signal void task_complete();
 
-	public AsyncTask(){
+	protected AsyncTask(){
 		working_dir = TEMP_DIR + "/" + timestamp_for_path();
 		script_file = path_combine(working_dir, "script.sh");
 		log_file = path_combine(working_dir, "task.log");
